pytorch/Dockerfile

```diff
@@ -80,11 +80,17 @@ RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missin
     gcc \
     libgl1-mesa-glx \
     libglib2.0-0 \
+    openssh-server \
```

Comment:
This package will generate an SSH key, you need to remove it because it will be flagged as a security vulnerability.

Reply:
we need ssh server. if openssh-server doesn't work, any other suggestion?

Reply:
Yes, exactly that. Install it and then `&& rm -rf /path/to/key` It must happen in the same `RUN`

Reply:
addressed accordingly